main.tf

```diff
@@ -126,15 +128,19 @@ locals {
     environment = local.environment
     stage       = local.stage
     # For AWS we need `Name` to be disambiguated since it has a special meaning
-    name       = local.id
-    attributes = local.id_context.attributes
+    name        = local.id
+    attributes  = local.id_context.attributes
+    root_module = local.root_module
   }
 
+  # Remove null values from tags_context. Required due to `root_module` that can be null.
+  tags_context_clean = { for k, v in local.tags_context : k => v if v != null }
```

Comment:
Hi @gberenice just a thought - if you set `root_module` to `""` instead of null (when it's not enabled), you don't need this line. Perhaps this would be more in line with the treatment of the other variables, as done when setting `normalized_labels`?